### ISSUE TITLE
Define the default template for template creation in theme.json

### ISSUE THREAD

**Issue by carolinan** [WordPress/gutenberg#42603]
## What problem does this address?

When users create a new custom page template from the block editor's Template section, WordPress uses a basic template with a few selected blocks.

## What is your proposed solution?

I know there is a lot of ongoing work to improve the template creation flow ❤️. It would be an additional improvement for theme developers if they could define a template html file to be used as the default.

This could be part of the customTemplates section in theme.json:
```

	"customTemplates": [
		{
			"name": "page-without-title",
			"title": "Page without title",
			"default": true
		}
	],

```

**Comment by Mamaduka** [comment]
Probably a duplicate of #36086, but in this case, I think we can close the old issue.

P.S. I like the idea of using the `default` flag in the definition.

**Comment by ellenbauer** [comment]
That would be so incredible helpful to have. It's not a great experience for users to have to switch the template from default every time. For our block themes, we would much rather have our Header & Footer only Template be the default one.

**Comment by carolinan** [comment]
Yes, it is the same issue, I could not find #36086 when searching :)

**Comment by carolinan** [comment]
I am not sure if this is needed anymore?
There is now a way to allow the user to select from patterns when they create a new template.
